.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-- 2.6
+- 2.6.0
```

Comment:
This change shouldn't be necessary.

Reply:
This change switched the Ruby version from 2.6.1 implicit which was not being found to 2.6.0 explicit which was. It was intentional and necessary as an alternative to switching to 2.5.